src/components/screens/monitor/blockDetails/blockDetails.js

```diff
@@ -125,6 +125,7 @@ const BlockDetails = ({
         title={t('Transactions')}
         transactions={blockTransactions}
         emptyStateMessage={t('There are no transactions for this block.')}
+        canLoadMore={false}
```

Comment:
It's an edge case, but the genesis block (height=1) has more than 30 transactions, so it might be worth to have that covered

Reply:
Thanks. I forgot about genesis block. Fixed it.